src/ui/public/state_management/state_hashing/unhash_query_string.js

```diff
@@ -0,0 +1,8 @@
+import { mapValues } from 'lodash';
+
+export default function unhashQueryString(parsedQueryString, states) {
```

Comment:
Yeah, I've seen that rule enforced in their eslint config as well. Like you said, the rule is based on the number of exports, which makes little to no sense to me. Importing code needs to know how many values you export? What if you decide to add a second import? Do you have to export a single value multiple ways now?

I think a lot of this stuff is so new that people are simply picking rules so they have a standard to follow. I would bet that this because rule was chosen because `module.exports =` and `export default` are very similar and `module.exports =` was the most common export style in CommonJS. This means that npm modules and app code are imported the same way, which I'm fine with.

On the contrary though, I would also bet that the `module.exports =` pattern beat out `exports.name =` because we didn't have dereferencing assignment. Now that we do, I almost always import the specific named methods of a module when I'm pulling it in (unless that module exports a single function).